README.md

```diff
@@ -22,80 +22,47 @@ Much like the contents of a container image need to be pulled and unpacked in or
 In this sense, Bundle is a generalization of the image concept, and can be used to represent any type of content.
 
 `Bundles` do nothing on their own - they require a `Provisioner` to unpack and make their content available in-cluster. They can be
-unpacked to any arbitrary storage medium such as a PersistentVolume or ConfigMap. Each `Bundle` has an associated `spec.class` field which 
+unpacked to any arbitrary storage medium such as a PersistentVolume or ConfigMap. Each `Bundle` has an associated `spec.provisionerClassName` field which 
 indicates the `Provisioner` that should be watching and unpacking that particular bundle type. 
 
 Example Bundle
 ```yaml
 apiVersion: core.rukpak.io/v1alpha1
 kind: Bundle
-metadata: 
-  name: example-operator.v0.9.3
+metadata:
+  name: combo-0.0.1
 spec:
-  class: rukpack.io/k8s
-  refs:
-  - file://content
-  volumeMounts:
-  - mountPath: /content
-    configMap:
-      name: local 
-      namespace: plumbus
+  image: quay.io/tyslaton/combo-plain-bundle:v0.0.1
```

Comment:
Mental note to come back here once we have a more official plain bundle format.

Reply:
Yes, ideally these example images would live in the `operator-framework` repository

Reply:
Going to use this image for now as it's the only semi-official plain+v0 out there right now